CONTRIBUTING.md

```diff
@@ -11,6 +11,15 @@ expectations the Node Foundation requires of its contributors.
 * A **Contributor** is any individual creating or commenting on an issue or pull request.
 * A **Collaborator** is a subset of contributors who have been given write access to the repository.
 
+## Start Here:
+
+- Take a look at any Issues tagged with [Good First Contribution](https://github.com/nodejs/community-committee/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+contribution%22)
+
+## Teams and Working Groups
+The Community Committee is a Committee that represents a collection of teams and working groups that are expressly working toward growing and supporting the overall Node.js community. These teams and working groups need a broad range of skills and allow you to offer contributions that are not necessarily technical in nature.
```

Comment:
"The Community Committee represents a collection of teams and working groups that are expressly working toward growing and supporting the overall Node.js community. Node.js as a whole needs a broad range of skills and contributions that are not necessarily technical in nature to thrive."

The teams and working groups don't need skills, and don't actually allow contributions. Tightened up the language a bit.